packages/storybook8/stories/Examples/CustomButtonInjection/Docs.mdx

```diff
@@ -0,0 +1,74 @@
+import * as CustomButtonInjectionStories from './index.stories';
+import { addCSS } from '../../utils';
+
+import CustomButtonWithAsyncStateExampleText from '!!raw-loader!./snippets/CustomButtonWithAsyncStateText.snippet.tsx';
+import CustomButtonWithStateExampleText from '!!raw-loader!./snippets/CustomButtonWithStateText.snippet.tsx';
+import LegacyControlBarCustomButtonInjectionExampleText from '!!raw-loader!./snippets/LegacyControlBarCustomButtonInjectionText.snippet.tsx';
+import MobileCustomButtonInjectionExampleText from '!!raw-loader!./snippets/MobileCustomButtonInjectionText.snippet.tsx';
+import OverflowCustomButtonInjectionExampleText from '!!raw-loader!./snippets/OverflowCustomButtonInjectionText.snippet.tsx';
+import PrimaryCustomButtonInjectionExampleText from '!!raw-loader!./snippets/PrimaryCustomButtonInjectionText.snippet.tsx';
+import SecondaryCustomButtonInjectionExampleText from '!!raw-loader!./snippets/SecondaryCustomButtonInjectionText.snippet.tsx';
+
+<Meta
+  of={CustomButtonInjectionStories}
+  component={CustomButtonInjectionComponent}
+/>
+
+<div id='custom-button-injection-story'>
+<script> { addCSS('#custom-button-injection-story button[aria-label="Custom"]{ border: 1px solid green; }') } </script>
+
+# Custom Button Injection
+
+Custom Button Injection in our Composite experiences allows users to add their own functional buttons. Providing
+options to build out their own custom experiences like components, we want to make sure injecting customer's own
+features is simple and achievable.
+
+# Examples
+
+There are multiple areas, `primary`, `secondary`, and `overflow`, that you can inject buttons into. Depending on
+which area, only a max number of custom injected buttons will be displayed. This also varies if you are mobile
+or desktop.
+
+In all the examples below we use a mock adapter to quickly showcase custom button injection in our Call
+Composite. Please ensure you replace the mock adapter with your own adapter.
+
+## Desktop: Primary Main Bar
+Max number of buttons: '3'
+
+<Canvas of={CustomButtonInjectionStories.PrimaryCustomButtonInjectionDocsOnly} source={{ code: PrimaryCustomButtonInjectionExampleText }} />
+
+## Desktop: Secondary Main Bar
+Max number of buttons: '2'
+
+<Canvas of={CustomButtonInjectionStories.SecondaryCustomButtonInjectionDocsOnly} source={{ code: SecondaryCustomButtonInjectionExampleText }} />
+
+## Desktop: Overflow More Button
+Max number of buttons: '∞'
+
+<Canvas of={CustomButtonInjectionStories.OverflowCustomButtonInjectionDocsOnly} source={{ code: OverflowCustomButtonInjectionExampleText }} />
+
+## Mobile: Primary and Overflow Contextual Menu
+Max number of buttons in primary: '1' Max number of buttons in Contextual Menu: '∞'
+
+<Canvas of={CustomButtonInjectionStories.MobileCustomButtonInjectionDocsOnly} source={{ code: MobileCustomButtonInjectionExampleText }} />
+
+## Custom Button With State
+Max number of buttons: '3'
```

Comment:
I don't see the text in the original storybook. Should this be there
![image](https://github.com/user-attachments/assets/a70baedb-e73c-473d-8218-1a78e6375501)

Reply:
Ah good call out! Thanks for looking back at the storybook v6 version to confirm if the string was originally there or not. This is technically not "wrong" but is a discrepancy between the other storybook and now. I will remove it anyways cause why not.